sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/setup/SetupAccountActivity.kt

```diff
@@ -82,24 +104,37 @@ class SetupAccountActivity : DuckDuckGoActivity(), SetupFlowListener {
     private fun processCommand(it: Command) {
         when (it) {
             Close -> finish()
+            RecoverSyncData -> loginFlow.launch(null)
         }
     }
 
     private fun renderViewState(viewState: ViewState) {
         when (viewState.viewMode) {
             AskSyncAnotherDevice -> {
+                screen = SETUP
                 supportFragmentManager.commitNow {
-                    replace(id.fragment_container_view, SyncSetupFlowFragment.instance(SyncAnotherDeviceScreen), TAG_RECOVER_ACCOUNT)
+                    replace(id.fragment_container_view, SyncSetupFlowFragment.instance(SyncAnotherDeviceScreen), TAG_ENABLE_SYNC)
```

Comment:
is this the correct tag?

Reply:
it is, probably wrong in the previous branch, will fix there.